packages/rum/test/e2e/general-usecase/app.js

```diff
@@ -111,4 +111,27 @@ generateError.tmp = 'tmp'
 
 testFetch(mockBackendUrl)
 
+if (location.hash === '#test-state') {
+  const path = location.pathname
+  history.pushState(
+    { data: 'test-state' },
+    'Push state title',
+    path + '#test-state'
+  )
+  history.pushState(
```

Comment:
is this intentional to call it multiple times?

Reply:
Yeah, because there's `go(-1)` call after that.

Reply:
Can you change the arguments to 2nd call in that case? To make sure we are actually testing the right thing.